src/cli.js

```diff
@@ -225,11 +239,12 @@ program
 program
   .command('generate-migration')
   .description('Generate a content migration file')
+  .option('-r, --region [value]', 'region', 'eu')
   .requiredOption('-c, --component <COMPONENT_NAME>', 'Name of the component')
   .requiredOption('-f, --field <FIELD_NAME>', 'Name of the component field')
   .action(async (options) => {
-    const field = options.field || ''
-    const component = options.component || ''
+    const { field } = options || ''
+    const { component } = options || ''
```

Comment:
Here, maybe the correct code would be:

```js
const { field = '' } = options
const { component = '' } = options
```

Because we can destruct a string, but the `field` and `component` will be undefined, instead of an empty string